### PR TITLE
chore: copy from Services `develop`

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
 - n/m

### CODE REVIEW

File: block/stream/output/smart_contract_service.proto [hunk 1]
```diff
@@ -158,4 +158,22 @@ message EthereumOutput {
...
+        proto.ContractFunctionResult ethereum_call_result = 3;
+
+        /**
+         * A result for an Ethereum Transaction executed as a create.
+         * <p>
+         * This field SHALL contain all of the data produced by the contract
+         * create transaction as well as basic accounting results.
+         */
+        proto.ContractFunctionResult ethereum_create_result = 4;
+    }
```
Comment (jsync-swirlds) [inline block/stream/output/smart_contract_service.proto:178]: This change is under some discussion in the Block Stream HIP, so @Nana-EC may wish to weigh in on it.
There are some other, related, changes to the Block Stream as well.

File: services/state/schedule/schedule.proto [hunk 2]
```diff
@@ -141,12 +141,10 @@ message Schedule {
...
-    /**
-     * a list of schedules, in no particular order.
-     * <p>
-     * While the order is not _specified_, it MUST be deterministic.
-     */
-    repeated Schedule schedules = 1;
+  /**
+   * a list of schedules, in no particular order.
+   */
+  repeated Schedule schedules = 1;
```
Comment (jsync-swirlds) [inline services/state/schedule/schedule.proto:147]: I would recommend reversing this specific change (change services to match).
The documentation in this repo is more clear, and the indentation is correct.

File: services/state/schedule/schedule.proto [hunk 3]
```diff
@@ -155,10 +153,40 @@ message ScheduleList {
  * One example is all schedules that expire at a particular time.
  */
 message ScheduleIdList {
+  /**
+   * A list of schedule identifiers, in no particular order.
+   * <p>
+   * While the order is not _specified_, it MUST be deterministic.
+   */
+  repeated ScheduleID schedule_ids = 1;
```
Comment (jsync-swirlds) [inline services/state/schedule/schedule.proto:161]: Indentation is off, should be 4 spaces.

File: services/state/schedule/schedule.proto [hunk 4]
```diff
@@ -155,10 +153,40 @@ message ScheduleList {
...
+}
+
+/**
+ * The value of a map summarizing the counts of scheduled and processed transactions
+ * within a particular consensus second.
```
Comment (jsync-swirlds) [inline services/state/schedule/schedule.proto:166]: Might need to be a different PR, but the specification text for this message and the next needs to be updated to specification standards rather than simple description.

File: services/state/tss/tss_status.proto [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+    /**
+     * The TSS keying process has completed and the ledger id is set.
+     */
+    KEYING_COMPLETE = 3;
+}
+
+/**
+ * An enum representing the key either active roster or candidate roster.
+ * This value will be to key active roster if it is genesis stage.
```
Comment (jsync-swirlds) [inline services/state/tss/tss_status.proto:99]:
```suggestion
 *
 * This value SHALL be to key active roster if it is genesis stage.
```

**Review by lpetrovic05 — APPROVED** [review]

**Review by iwsimon — APPROVED** [review]
LGTM, thanks @tinker-michaelj

File: block/stream/output/smart_contract_service.proto (hunk 1 above)
Comment (tinker-michaelj) [reply block/stream/output/smart_contract_service.proto:178]: Created [this issue](https://github.com/hashgraph/hedera-services/issues/16991) to address the suggestions in `hedera-services` repo. 👍
